frontend/src/main/components/Jobs/InstructorReportJobForm.js

```diff
@@ -0,0 +1,62 @@
+import { Button, Form } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+
+function InstructorReportJobForm({ submitAction }) {
+
+ const defaultValues = {
+    // Stryker disable next-line all
+    fail: false,
+    sleepMs: 1000
```

Comment:
These parameters serve no purpose for the InstructorReportForm.  

Please remove these.   The same will be true of all of the other jobs except for the TestJob.  That is the only one of the four jobs that has any parameters.